Source/WebCore/page/FrameTree.h

```diff
@@ -75,13 +75,16 @@ class FrameTree {
     WEBCORE_EXPORT unsigned childCount() const;
     unsigned descendantCount() const;
     WEBCORE_EXPORT AbstractFrame& top() const;
+    unsigned depth() const;
 
     WEBCORE_EXPORT AbstractFrame* scopedChild(unsigned index) const;
     WEBCORE_EXPORT AbstractFrame* scopedChild(const AtomString& name) const;
     unsigned scopedChildCount() const;
 
     void resetFrameIdentifiers() { m_frameIDGenerator = 0; }
 
+    static const int maxFrameDepth = 32;
```

Comment:
`constexpr`

Not sure this class is the best place for this limit, but I don’t have a strong opinion about a better place.

Reply:
I think this and Page::maxNumberOfFrames should be in the same place.